debug/debug_state_separator.py

```diff
@@ -9,7 +9,7 @@
 import webgui.lattice_view
 from fractions import Fraction
 
-import qiskit.aqua.operators as qk
+import qiskit.opflow as qk
```

Comment:
We are both using `qiskit.opflow.*` and `qk.*` in our code - can you unify them under 1 format? I am thinking `qkop` or something similar.